homework12/project_django/project_orm/get_report.py

```diff
@@ -0,0 +1,20 @@
+import csv
+import os
+
+import django
+
+
+def main():
```

Comment:
You can register custom script as manage.py command.  
https://docs.djangoproject.com/en/3.2/howto/custom-management-commands/